test/ScriptCs.Tests.Acceptance/ScriptLibraries.cs

```diff
@@ -72,7 +72,7 @@ public static void LoadingFromANonRootedScript(ScenarioDirectory directory, Scen
                 .f(() => ScriptCsExe.Install("ScriptCs.Calculator", scriptDirectory));
 
             "When executing the script"
-                .f(() => output = ScriptCsExe.Run(Path.Combine("script", "foo.csx"), directory));
+                .f(() => output = ScriptCsExe.Run(Path.Combine("script", "foo.csx"), false, directory));
```

Comment:
I guess this mean the execution of scripts which use script libraries fails when running under `-debug`?